modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -576,7 +576,8 @@ public void processOpts() {
             // The flag below should be set for all Java libraries, but the templates need to be ported
             // one by one for each library.
             supportsAdditionalPropertiesWithComposedSchema = true;
-
+            setUseJakartaEe(true);
```

Comment:
What is the plan to expose the option to users via the cli? SpringCodegen has the useSpringBoot3 indirection, which triggers the use of Jakarta EE.

Reply:
as for now this would be internal config option. later I want to use this config for changes like this one - https://github.com/OpenAPITools/openapi-generator/issues/14276

`rest-template` to be migrated to spring 6 which means `jakarta` should be used, but on the other hand `rest-template` shares common `model.mustache` with other libraries which still should use `javax` package.

that's the reason why this config option is internal - developer of library's template is aware whether `javax` or `jakarta` package to be used

Reply:
Ok, got it. But the default should then be false `setUseJakartaEe(false)`, otherwise this PR would activate jakarta EE for all JavaClientCodegen implementations.

Reply:
ahhh. got you question) expand this part of code and you will notice that this is only for jersey 3 which already use `jakarta` package. by default this option is set to `false` (i.e. we are using `javax` package)